Makefile

```diff
@@ -39,55 +39,55 @@ functional: ginkgo
 ifeq (${RUNTIME},)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	chronic ./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
```

Comment:
How about doing something like this:

```
ifeq ($(V),)
    PRECMD=@chronic
endif

ifeq ($(V),1)
    PRECMD=@
endif

   :

$(PRECMD) ./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
```

Reply:
Yep, that'd work. I did wonder if we might want some (default on) quiet mode. I'll see what the CIs think, and garner any further input - and then very likely make that change.